README.md

```diff
@@ -32,7 +46,7 @@ To control the format of the exported RDF, the extension and RDF language/serial
 --rdfExt .jsonld --rdfLang application/ld+json
 ```
 
-The list of RDF languages supported can be found in the [RESTful HTTP API documentation](application/ld+json).
+The list of RDF languages supported can be found in the [RESTful HTTP API documentation](https://wiki.duraspace.org/display/FEDORA4x/RESTful+HTTP+API).
```

Comment:
Since the serialization formats will likely not be changing much at this point, listing them is probably reasonable.